src/Provider/Keycloak.php

```diff
@@ -134,6 +134,30 @@ public function getResourceOwnerDetailsUrl(AccessToken $token)
         return $this->getBaseUrlWithRealm().'/protocol/openid-connect/userinfo';
     }
 
+    /**
+     * Builds the logout URL.
+     *
+     * @param array $options
+     * @return string Authorization URL
+     */
+    public function getLogoutUrl(array $options = [])
+    {
+        $base = $this->getBaseLogoutUrl();
+        $params = $this->getAuthorizationParameters($options);
+        $query = $this->getAuthorizationQuery($params);
+        return $this->appendQuery($base, $query);
+    }
+
+    /**
+     * Get logout url to logout of session token
+     *
+     * @return string
+     */
+    public function getBaseLogoutUrl()
```

Comment:
Is there ever a use case where someone who need to retrieve the base logout url (without authorization) instead of the logout url (with authorization)? If so, I'd recommend making this method private, so it functions as a helper method and will not "confuse" the developer if the authorized logout url is the only one that really matters.